apps/shared/avifjpeg.c

```diff
@@ -354,24 +380,133 @@ avifBool avifJPEGRead(const char * inputFilename,
         }
     }
     if (!ignoreXMP) {
-        const avifROData tagStandardXmp = { (const uint8_t *)"http://ns.adobe.com/xap/1.0/\0", 29 };
-        const avifROData tagExtendedXmp = { (const uint8_t *)"http://ns.adobe.com/xmp/extension/\0", 35 };
-        avifBool found = AVIF_FALSE;
+        const avifROData tagStandardXMP = { (const uint8_t *)"http://ns.adobe.com/xap/1.0/\0", 29 };
```

Comment:
Nit/optional: `\0` can be omitted in this string and in the string in line 400. We can use the terminating NUL bytes in string literals.

Reply:
I believe this is clearer this way, without a comment mentioning that the hard-coded length includes the implicit `\0`.